scripts/scripts/vaccinations/src/vax/incremental/ireland.py

```diff
@@ -63,7 +63,7 @@ def enrich_location(ds: pd.Series) -> pd.Series:
 
 
 def enrich_vaccine(ds: pd.Series) -> pd.Series:
-    return enrich_data(ds, 'vaccine', "Moderna, Oxford/AstraZeneca, Pfizer/BioNTech")
+    return enrich_data(ds, 'vaccine', "Johnson&Johnson, Moderna, Oxford/AstraZeneca, Pfizer/BioNTech")
```

Comment:
No evidence they are reporting this in their [official dashboard](https://covid19ireland-geohive.hub.arcgis.com/).

Unless found, should be

```python
return enrich_data(ds, 'vaccine', "Moderna, Oxford/AstraZeneca, Pfizer/BioNTech")
```